docs/usage-tui.md

```diff
@@ -0,0 +1,207 @@
+
+    kill <1>
+
+You can also make some commands depend on others by using `wait` command with a specific task id.
+
+### User interface and Knit integration
+
+Some of the `Knit` functions may access the internal state of Ariadne UI &mdash; namely, currently
+selected wallet, account or address. This allows you to omit the wallet name when creating a new
+account using `new-account` command &mdash; it will use the currently selected one. There is also a
+`copy-selection` command, which will copy the name of selected item (wallet, account or address) to
+the system clipboard.
+
+### Cardano-related commands
+
+Finally, to the commands which will help you manage your `ADA` wallet. Each command will be
+annotated with its arguments and their types. Optional arguments will be marked with "`?`".
+
+- `new-wallet pass: String? name: String? entropy-size: Int?`: create a new wallet with a
+    passphrase, name and size of entropy. Size of entropy influences security of your wallet and the
+    length of the mnemonic. Default value is 16 bytes, leading to mnemonics of 13 words.
+- `new-account wallet: String or Word name: String?`: create a new account in the specified or
+    the currently selected wallet. You can specify wallet either by its name or by its (zero-based)
+    index in the wallet tree.
+- `new-address wallet: String or Word account: String or Word pass: String?`: create a new address
+    with an optional passphrase. This command will use either the currently selected account or the
+    one specified by `wallet` and `account` arguments. Once again, you can either use names or
+    indices of wallets and accounts.
+- `balance`: get balance of the currently selected wallet, account or address.
+- `select wallet: String or Word a: Word? a: Word?`: select an item. Specify a wallet by its name or
+    index and then, optionally, specify indices of an acccount in that wallet and address in that
+    account. Like this: `select "my wallet" 1 0` to select the first address in the second account.
+- `rename name: String`: give a new name to the currently selected wallet or account.
+- `remove`: remove currently selected wallet, account or address.
+- `restore pass: String? name: String? mnemonic: String full: Bool`: restore a wallet from mnemonic.
+    A passphrase is used to encrypt the restored wallet in Ariadne, it does not have to be the same
+    as the passphrase the old wallet was encrypted with. `full` argument specifies whether Ariadne
+    should perform a full restore: find all used accounts and addresses of the restored wallet in
+    the blockchain.
+- `restore-from-daedalus-file name: String? file: FilePath full: Bool`: restore wallet from
+    Daedalus's secret file. `full` argument has the same effect as for `restore` command. Please
+    note that the restored wallet will have the same passphrase as in Daedalus.
+
+There is also a command to send a transaction from you wallet: `send`. Since one transaction can
```

Comment:
Why is `send` so special that you described it separately and didn't provide list of its argument in common notation used above? :thinking:

Reply:
s/you/your

Reply:
Because it's more complicated than a simple command with clear signature. Refactored it a bit.